Fixed issue #649 Broken Link

Anishka0107 opened the pull request:
This pull request tries to fix the issue https://github.com/openaustralia/righttoknow/issues/649
The link Statement of IP Principles for Australian Government Agencies on the page https://www.righttoknow.org.au/help/requesting#reuse is broken. It initially pointed to a page which does not exist i.e., https://www.ag.gov.au/RightsAndProtections/IntellectualProperty/Documents/StatementofIPprinciplesforAusGovagencies.pdf
Now it links to the page http://www.ags.gov.au/publications/express-law/el125.pdf

Anishka0107 committed:
Fixed issue #649 Broken Link

equivalentideas commented:
Thanks @Anishka0107 

I'm a little surprised there's nothing more up to date that 2010 for their latest statement, so I did a little searching and found this https://www.communications.gov.au/policy/policy-listing/australian-government-intellectual-property-rules. Is the PDF linked to their relevant?

We'll also need to update the quote below the link with something from the new principles that makes the same point.

equivalentideas reviewed (changes requested):
* [ ] Check if this document is the more up-to-date statement https://www.communications.gov.au/policy/policy-listing/australian-government-intellectual-property-rules
* [ ] Update the quote below to something that makes the same point from the new document.

equivalentideas commented:
I've just added those steps as a review on GitHub :)

Anishka0107 commented:
@equivalentideas 
After doing a bit of research, I got the document which includes the encouragement of re-use.
It is http://www.finance.gov.au/sites/default/files/Government20TaskforceReport.pdf?v=1
Could you please navigate to page number 21 and read recommendation 6.3
This is the right document I guess. I have also committed it.

Anishka0107 committed:
Updated dead link of reuse encouragement

This commit aims to fix the issue https://github.com/openaustralia/righttoknow/issues/649
The page https://www.righttoknow.org.au/help/requesting#reuse contains a dead link of the Statement of IP Principles of Australian Government Agencies, which points to https://www.ag.gov.au/RightsAndProtections/IntellectualProperty/Documents/StatementofIPprinciplesforAusGovagencies.pdf.
It has now been updated to the link http://www.finance.gov.au/sites/default/files/Government20TaskforceReport.pdf?v=1 which contains the above information under Recommendation 6.3 of the Government 2.0 Taskforce Report. 
Also, the statement in the block has been modified to that present in the above report.

equivalentideas commented:
I think @benrfairless might be the best person to review this. I'm going to assign you Ben, but feel free to throw it back at me :D 

Thanks @Anishka0107 !

benrfairless reviewed (changes requested):
I still think that the correct file is https://www.communications.gov.au/file/20151/download?token=nmvWmdrj, which I got from https://www.communications.gov.au/policy/policy-listing/australian-government-intellectual-property-rules. This are the actual rules in effect - not recommendations made.

mlandauer commented:
Looking at the most recent comment from @ben it looks like the now the most recent relevant document is most likely the "Intellectual property principles for Commonwealth entities" updated Feb 2019 available at https://www.communications.gov.au/documents/intellectual-property-principles-commonwealth-entities.

It says in section 11(b):
"Consistent with the need for free and open re-use and adaptation, public sector information should be licensed by entities under the Creative Commons BY standard as the default."

mlandauer commented:
I've applied the fix in https://github.com/openaustralia/righttoknow/commit/d4064088cdf464ec7f6bddfcf459059c42a80e4e. Closing this PR in favour of that change
